# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/util/ChromeProcessController.h
# Commit: febdcf7986a3
# Full Hash: febdcf7986a37651188476cd8253d20e9cea443c
# Author: Mike Conley <mconley@mozilla.com>
# Date: 2022-06-23 03:54:50
# Regressor Bug: 1773865
# File Overlap Count: 1
# Description:
#   Bug 1773865 - Dispatch an event on the window document when a pinch zoom gesture ends. r=botond,smaug,NeilDeakin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D149283
# ==============================================================================

diff -r 00d1c650d88f -r febdcf7986a3 gfx/layers/apz/util/ChromeProcessController.h
--- a/gfx/layers/apz/util/ChromeProcessController.h	Wed Jun 22 19:26:58 2022 +0000
+++ b/gfx/layers/apz/util/ChromeProcessController.h	Wed Jun 22 19:40:17 2022 +0000
@@ -75,6 +75,8 @@
   void NotifyAsyncAutoscrollRejected(
       const ScrollableLayerGuid::ViewID& aScrollId) override;
   void CancelAutoscroll(const ScrollableLayerGuid& aGuid) override;
+  void NotifyScaleGestureComplete(const ScrollableLayerGuid& aGuid,
+                                  float aScale) override;
 
   PresShell* GetTopLevelPresShell() const override { return GetPresShell(); }
 